spec/rails_helper.rb

```diff
@@ -100,6 +100,10 @@
     )
   end
 
+  config.before(:each, :might_call_cfe) do
```

Comment:
How come you can't use the `stub_cfe_calls`. Why create a `might_call_cfe` stub? 

Also `might_call_cfe` isn't a clear and very confident name, something that describes it what it does would be better.

If they are both necessary then can we make it clear how they are different from each other and used.

Reply:
Agreed I couldn't think of a good name. I wanted to convey that this is to do with early eligibility - thought of that over the weekend. Will find a better name